content/cockpit-129.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 128 and 129
+author: Dominik Perpeet
+date: 2016-01-19 09:00
+tags: cockpit, linux
+slug: cockpit-129
+category: release
+summary: Cockpit supports Rebases and Remotes in OSTree.
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release
+regularly. Here are the release notes from version 128 and 129.
+
+### Manage remotes and rebase in OSTree
+
+When working with [OSTrees](http://www.projectatomic.io/docs/os-updates/) there will
+often be multiple branches to choose from. Thanks to Peter's work, Cockpit can now switch
```

Comment:
often be multiple branches to choose from. "For example there may be a beta version of the operating system." Thanks to Peter's ...